doc/combined_analysis.md

```diff
@@ -85,7 +85,7 @@ First of all create a foder to put the final data about EoS:
 
 Then run the last one command line
 
-	 combined_EOS --outdir ./output/Figures --label ZTF --gwR14trend ./example_files/ --GWEMsamples ./output/GW_EMdata --detections-file ./example_files/csv_lightcurve/detectable.txt --EOS-prior ./example_files/eos/EOS_sorted_weight.dat --EOSpath ./example_files/eos/eos_sorted  --pdet ./example_files/eos/pdet_of_Mmax.dat --R14_true 11.55 --Neos 5000 --seed 42  --cred-interval 0.95
+	 combined_EOS --outdir ./output/Figures --label ZTF --gwR14trend ./example_files/eos/GW_R14trend.dat --GWEMsamples ./output/GW_EMdata --detections-file ./example_files/ztf_ligthcurve_detectable/detectable.txt --EOS-prior ./example_files/eos/EOS_sorted_weight.dat --EOSpath ./example_files/eos/eos_sorted  --pdet ./example_files/eos/pdet_of_Mmax.dat --R14_true 11.55 --Neos 5000 --seed 42  --cred-interval 0.95
```

Comment:
ztf_ligthcurve_detectable -> ztf_lightcurve_detectable 
maybe this also needs to be fixed in the repo?

Reply:
Yes I will add some details about the provenance   on doc.

ok  ztf_lightcurve_detectable, 

I fixed it  , Thanks,